example/lib/circle_annotations_example.dart

```diff
@@ -55,11 +55,10 @@ class CircleAnnotationExampleState extends State<CircleAnnotationExample> {
         ));
       }
       circleAnnotationManager?.createMulti(options);
-      circleAnnotationManager?.addOnCircleAnnotationClickListener(
-        AnnotationClickListener(
-          onAnnotationClick: (annotation) => circleAnnotation = annotation,
-        ),
-      );
+      circleAnnotationManager?.tapEvents(onTap: (annotation) {
```

Comment:
I think we can now remove the AnnotationClickListener class from this example (and the other examples). It would be good to do this so devs are not confused.